release-notes/6.0/api-diff/.Net/6.0.0_System.Numerics.md

```diff
@@ -0,0 +1,72 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
+-    public readonly struct BigInteger : IComparable, IComparable<BigInteger>, IEquatable<BigInteger>, IFormattable
++    public readonly struct BigInteger : IComparable, IComparable<BigInteger>, IEquatable<BigInteger>, IFormattable, ISpanFormattable
+     public static class BitOperations {
++        public static bool IsPow2(int value);
++        public static bool IsPow2(long value);
++        public static bool IsPow2(uint value);
++        public static bool IsPow2(ulong value);
++        public static uint RoundUpToPowerOf2(uint value);
++        public static ulong RoundUpToPowerOf2(ulong value);
+     }
+-    public struct Complex : IEquatable<Complex>, IFormattable
++    public readonly struct Complex : IEquatable<Complex>, IFormattable
+     public static class Vector {
++        public static Vector<TTo> As<TFrom, TTo>(this Vector<TFrom> vector) where TFrom : struct where TTo : struct;
++        public static Vector<IntPtr> AsVectorNInt<T>(Vector<T> value) where T : struct;
```

Comment:
Just wanted to explicitly call out @Anipik. This says `IntPtr` and isn't accounting for the difference that it's really `nint` (they are the same type, but how they work is subtly different in the language).

Reply:
Same goes for `UIntPtr` below in `AsVectorNUInt` and a few similar APIs in my areas

Reply:
do u know in which preview they were first introduced ?